README.md

```diff
@@ -68,14 +68,7 @@ If you do not want any data to be gathered, you can turn off this feature by set
 ## Help Needed on [Desktop Version](https://github.com/PokemonGoF/PokemonGo-Bot-Desktop)
 
 
-## Credits
-- [tejado](https://github.com/tejado) many thanks for the API
-- [U6 Group](http://pgoapi.com) for the U6
-- [Mila432](https://github.com/Mila432/Pokemon_Go_API) for the login secrets
-- [elliottcarlson](https://github.com/elliottcarlson) for the Google Auth PR
-- [AeonLucid](https://github.com/AeonLucid/POGOProtos) for improved protos
-- [AHAAAAAAA](https://github.com/AHAAAAAAA/PokemonGo-Map) for parts of the s2sphere stuff
-- [breeze ro] for some of that map stuff
+## [Credits](https://github.com/PokemonGoF/PokemonGo-Bot/blob/master/CONTRIBUTORS.md)
```

Comment:
aren't they in the credits? What is the difference in credits and contribution anyway?